nerdlandbot/commands/recipe.py

```diff
@@ -0,0 +1,140 @@
+import typing
+import discord
+from discord.ext import commands
+import gspread
+import os
+import asyncio
+from datetime import datetime
+
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.constants import NOTIFY_EMBED_COLOR
+
+SHEETS_TOKEN = os.getenv("SHEETS_JSON")
+SPREADSHEET = os.getenv("SPREADSHEET")
+
+
+class Recipe(commands.Cog, name="Spreadsheets"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+    
+
+    @commands.command(name="add_recipe", aliases=["recipe"], brief="recipe_brief", help="recipe_help")
+    async def add_recipe(self, ctx: commands.Context):
+        # Getting everything ready to acces 
+        gc = gspread.service_account(SHEETS_TOKEN)
+        sh = gc.open(SPREADSHEET)
+        ws = sh.sheet1
+        next_row = next_available_row(ws)
+
+        # Fetching date and formatting it
+        d_obj = datetime.now()
+        date_string = "{}/{}/{} {}:{}:{}".format(d_obj.month, d_obj.day, d_obj.year, d_obj.hour, d_obj.minute, d_obj.second)
+
+        # Initializing variables needed
+        lang = await culture(ctx)
+        embed_title = translate("recipe_title", lang)
+        questions = []
+        answers = []
+
+
+        
+        # Asking the user questions and capturing the answer
+        for i in range(5):
+            questions.append(translate("recipe_template", lang).format(translate("recipe_{}_question".format(i+1), lang)))
+            embed = discord.Embed(
+                title = embed_title,
+                description = questions[i],
+                color = NOTIFY_EMBED_COLOR
+            )
+            await ctx.send(embed=embed)
+            try:
+                reaction = await ctx.bot.wait_for("message", timeout=30, check=check(ctx.author))
+                await asyncio.sleep(1)
+                answers.append(reaction)
+                if i == 2:
+                    try:
+                        reaction_int = int(reaction.content)
+                        if reaction_int > 5 or reaction_int < 1:
+                            rating_error = translate("recipe_rating_error", lang)
+                            embed = discord.Embed(
+                                title = embed_title,
+                                description = rating_error,
+                                color = NOTIFY_EMBED_COLOR
+                            )
+                            return await ctx.send(embed=embed)
```

Comment:
It would be nice if you can just add a new number after this error instead of quitting the whole thing.